images/aws_upload.sh

```diff
@@ -64,6 +64,7 @@ echo "Making AMI public"
 aws ec2 modify-image-attribute --image-id "${ami_id}" --launch-permission "Add=[{Group=all}]"
 
 echo "AMI Created: ${ami_id}"
+echo "${ami_id}" >> ami_id.txt
```

Comment:
mmh maybe we should include AMI's from other regions too?

Reply:
ummm, yeah, that rigth, should probably be a list of `ami_id, region` instead of a simple ami id